### PR TITLE
Restructure robotlibcore into smalle chunks

### PR DESCRIPTION
**Pull request by rasjani**

### CODE REVIEW

**Comment by rasjani** [comment]
feedback welcome.

Known issues; inv print and set version tasks do not work at the moment.

File: setup.py [hunk 1]
```diff
@@ -21,8 +21,10 @@
 Topic :: Software Development :: Testing
 Framework :: Robot Framework
 """.strip().splitlines()
-with open(join(CURDIR, 'src', 'robotlibcore.py')) as f:
+
+with open(join(CURDIR, 'src', 'robotlibcore','__init__.py')) as f:
```
Comment (rasjani) [inline setup.py:25]: could be made to use pathlib

File: src/robotlibcore/core/dynamic.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import inspect
```
Comment (rasjani) [inline src/robotlibcore/core/dynamic.py:1]: missing copyright header

File: src/robotlibcore/core/hybrid.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import inspect
```
Comment (rasjani) [inline src/robotlibcore/core/hybrid.py:1]: missing copyright header

File: src/robotlibcore/keywords/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+from .builder import KeywordBuilder
```
Comment (rasjani) [inline src/robotlibcore/keywords/__init__.py:1]: missing copyright header

File: src/robotlibcore/keywords/builder.py [hunk 5]
```diff
@@ -0,0 +1,134 @@
+import inspect
```
Comment (rasjani) [inline src/robotlibcore/keywords/builder.py:1]: missing copyright header

File: src/robotlibcore/keywords/specification.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+class KeywordSpecification:
```
Comment (rasjani) [inline src/robotlibcore/keywords/specification.py:1]: missing copyright header

File: src/robotlibcore/plugin/parser.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
+import inspect
```
Comment (rasjani) [inline src/robotlibcore/plugin/parser.py:1]: missing copyright header

File: src/robotlibcore/utils/exceptions.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+class PythonLibCoreException(Exception):  # noqa: N818
```
Comment (rasjani) [inline src/robotlibcore/utils/exceptions.py:1]: missing copyright header

File: src/robotlibcore/__init__.py [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+"""Generic test library core for Robot Framework.
+
+Main usage is easing creating larger test libraries. For more information and
+examples see the project pages at
+https://github.com/robotframework/PythonLibCore
+"""
+
+from robot.api.deco import keyword
+
+from robotlibcore.core import DynamicCore, HybridCore
```
Comment (rasjani) [inline src/robotlibcore/__init__.py:24]: probably only DynamicCore, HybridCore, keyword & exceptions need to be exported from here

File: setup.py (hunk 1 above)
Comment (aaltat) [reply setup.py:25]: yes it could be.

File: src/robotlibcore/core/hybrid.py (hunk 3 above)
Comment (aaltat) [reply src/robotlibcore/core/hybrid.py:1]: Good catch, good catch.

File: src/robotlibcore/keywords/__init__.py (hunk 4 above)
Comment (aaltat) [reply src/robotlibcore/keywords/__init__.py:1]: True, please add.

File: src/robotlibcore/__init__.py (hunk 9 above)
Comment (aaltat) [reply src/robotlibcore/__init__.py:24]: Most likely

**Comment by aaltat** [comment]
Looks good, please add the missing copyright and then it’s ready to go

File: src/robotlibcore/__init__.py (hunk 9 above)
Comment (rasjani) [reply src/robotlibcore/__init__.py:24]: unittests do use some of these so i'll leave them as for now.

**Pull request merged**